### PR TITLE
added shared functionality to news items

### PR DESCRIPTION
**Pull request by Chiz-nn**
I have put a generic sharenews button in each news element. This  allows the user to share individual news stories

### CODE REVIEW

File: pages/news.js [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
   return (
     <View>
       <ScrollView>
         <Text style={globalStyles.newsTitle}>{route.params.title}</Text>
         <Text style={globalStyles.newsParagraph}>{route.params.dateString} | {route.params.author}</Text>
         <Text style={globalStyles.newsParagraph} dataDetectorType="all" >{route.params.story}</Text>
       </ScrollView>
+      <TouchableOpacity style={globalStyles.shareBtn} onPress={()=>testShare(route.params.story)}>
+            <Text style={globalStyles.shareTxt}>SHARE</Text>
+          </TouchableOpacity>
```
Comment (dr28-calvin) [inline pages/news.js:34]: y=Your indenting looks kinda odd.
```suggestion
        <Text style={globalStyles.shareTxt}>SHARE</Text>
      </TouchableOpacity>
```

File: pages/news.js [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
+      title:`Test Story Share`,
+      message:story,
+    });
+
+    if (result.action === Share.sharedAction) {
+      alert("Post Shared")
+    } else if (result.action === Share.dismissedAction) {
+      // dismissed
+      alert("Post cancelled")
+    }
```
Comment (dr28-calvin) [inline pages/news.js:17]: Don't know where the alert was supposed to show up. I did notice an alert once  it didn't consistently show up.

**Comment by dr28-calvin** [comment]
It just shares the body text of the story, and not the title or author which are also useful pieces of information.

File: pages/news.js (hunk 2 above)
Comment (dr28-calvin) [reply pages/news.js]: Still relevant.

**Comment by Nmad7** [comment]
Chiz you need use eslint. A lot of the problems can be solved by just merging with master as well.

**Review by dr28-calvin — APPROVED** [review]

**Pull request merged**